db/seeds.rb

```diff
@@ -5,29 +5,33 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-Page.create(title: 'Rólunk', name: 'about', content: 'Lorem ipsum dolor set')
-Page.create(title: 'Szabályok', name: 'rules', content: 'Lorem ipsum dolor set')
-Page.create(title: 'Jelentkezés', name: 'teams', content: 'Lorem ipsum dolor set')
-Page.create(title: 'Kölcsönzés', name: 'rents', content: 'Lorem ipsum dolor set')
+pgs = [['Rólunk', 'about'],
```

Comment:
Irjuk ki a változók neveit, könnyeben olvasható tőle kód. Legyen a változó neve `pages`.

Tetszik ez a listás megoldás,  a ruby-ban az ilyesmit hash-ekkel is meg lehet oldani. Az valahogy igy névne ki:
 ```ruby
pages = [{title:'Rólunk' , name:'about' },{title:'Szabályok' , name:'rules' },...]
pages.each{|page| Page.create!(page.merge content: 'Lorem ipsum dolor set')}
```